packages/wallet/src/KeyManagement/LedgerKeyAgent.ts

```diff
@@ -0,0 +1,217 @@
+import { Cardano, NotImplementedError } from '@cardano-sdk/core';
+import {
+  CommunicationType,
+  GroupedAddress,
+  KeyAgentType,
+  SerializableLedgerKeyAgentData,
+  SignBlobResult,
+  TransportType
+} from './types';
+import { KeyAgentBase } from './KeyAgentBase';
+import { TransportError } from './errors';
+import AppAda, { GetVersionResponse, utils } from '@cardano-foundation/ledgerjs-hw-app-cardano';
```

Comment:
```suggestion
import DeviceConnection, { GetVersionResponse, utils } from '@cardano-foundation/ledgerjs-hw-app-cardano';
```
All other properties are called `deviceConnection`, so why not use this as the import too?